### DISCUSSION TITLE
客户端grpc方式获取bornTimestamp（客户端消息发送时间）是utc时间

### DISCUSSION THREAD

**Discussion by asjfoajs** [apache/rocketmq#7990] [Q&A]
小白请教一下 我用grpc的方式发送消息，返回的bornTimestamp是utc时间，但我看dashbord 上的bornTimestamp的时间显示的是北京时间，这是和通信的方式不一样吗？grpc的序列化好像是utc时间？ dashbord是json序列化吗？

**Comment by leizhiyuan** [comment]
可以提供一下具体的展示效果。诣极bornTime的信息
